crates/store/src/key/alias.rs

```diff
@@ -0,0 +1,106 @@
+use std::convert::Infallible;
+
+#[cfg(feature = "borsh")]
+use borsh::{BorshDeserialize, BorshSerialize};
+use calimero_primitives::alias::{Alias as AliasPrimitive, Kind as KindPrimitive};
+use calimero_primitives::context::ContextId;
+use generic_array::sequence::Concat;
+use generic_array::typenum::{U1, U32, U50};
+use generic_array::GenericArray;
+
+use super::component::KeyComponent;
+use super::{AsKeyParts, FromKeyParts, Key};
+use crate::db::Column;
+
+#[derive(Clone, Copy, Debug)]
+pub struct Kind;
+
+impl KeyComponent for Kind {
+    type LEN = U1;
+}
+#[derive(Clone, Copy, Debug)]
+pub struct Scope;
+
+impl KeyComponent for Scope {
+    type LEN = U32;
+}
+
+#[derive(Clone, Copy, Debug)]
+pub struct Alias;
+
+impl KeyComponent for Alias {
+    type LEN = U50;
+}
+
+#[derive(Clone, Copy, Debug, Eq, Ord, PartialEq, PartialOrd)]
+#[cfg_attr(feature = "borsh", derive(BorshSerialize, BorshDeserialize))]
+pub struct IdentityAlias(Key<(Kind, Scope, Alias)>);
```

Comment:
this shouldn't be named `IdentityAlias` since it may represent context, identity or application

Reply:
```suggestion
pub struct Alias(Key<(Kind, Scope, Name)>);
```